Demos/benchmarks/run_benchmarks.py

```diff
@@ -251,12 +269,15 @@ def benchmark_revision(revision, benchmarks, cythonize_args=None, profiler=None,
 
         bm_dir.mkdir(parents=True)
         bm_files = copy_benchmarks(bm_dir, benchmarks)
+        sizes = None
         if not plain_python:
             compile_benchmarks(cython_dir, bm_files, cythonize_args, c_macros=c_macros)
+            if show_size:
+                sizes = measure_benchmark_sizes(bm_files)
 
         logging.info(f"### Running benchmarks for {revision}.")
         pythonpath = cython_dir if plain_python else None
-        return run_benchmarks(bm_dir, benchmarks, pythonpath=pythonpath, profiler=with_profiler)
+        return run_benchmarks(bm_dir, benchmarks, pythonpath=pythonpath, profiler=with_profiler), sizes
```

Comment:
If we give a name to one, let's take the opportunity to give names to both values.
```suggestion
        timings = run_benchmarks(bm_dir, benchmarks, pythonpath=pythonpath, profiler=with_profiler)
        return timings, sizes
```